src/topology/subset_properties.lean

```diff
@@ -230,6 +230,19 @@ lemma is_compact.disjoint_nhds_set_right {l : filter α} (hs : is_compact s) :
   disjoint l (𝓝ˢ s) ↔ ∀ x ∈ s, disjoint l (𝓝 x) :=
 by simpa only [disjoint.comm] using hs.disjoint_nhds_set_left
 
+/-- For every family of closed sets whose intersection avoids a compact set,
+there exists a finite subfamily whose intersection avoids this compact set. -/
```

Comment:
The docstring does not really correspond to the statement -- you should mention `directed` somewhere.